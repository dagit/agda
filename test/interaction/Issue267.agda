module Issue267 where

data I : Set where
  c : I

postulate
  T : I → Set
  P : ∀ i → T i → Set
  R : ∀ i → (T i → Set) → Set
  r : ∀ i → R i (P i)

P′ : ∀ i → T i → Set
P′ c x = P c x

foo : ∀ i → R i (λ v → P′ i v)
foo c = ?
