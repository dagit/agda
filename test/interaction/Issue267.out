agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*All Goals*"
                 "?0 : R c (λ v → P′ c v)
")
agda2_mode_code (agda2-goals-action '(0))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Goal type etc.*"
                 "Goal: R c (λ v → P c v)
Have: R c (P c)
————————————————————————————————————————————————————————————")
agda2_mode_code (agda2-goals-action '(0))
